auto_py_to_exe/web/css/general.css

```diff
@@ -1,6 +1,6 @@
 @font-face {
-    font-family: 'Nunito';
-    src: url('../Nunito-Light.ttf');
+    font-family: 'Ubuntu';
+    src: url('../Fonts/Ubuntu-Regular.ttf');
```

Comment:
Could we make the `Fonts` folder `fonts` - just to keep consistent. Also, I noticed you are only using the regular ttf file, so we don't need the other two.

Reply:
Will do.